### ISSUE TITLE
Installation failes on x64 windows 10. There must be only one argument: DEST_DIR (you gave C:\Users\Gaurav Jain\.ebcli-virtual-env) Usage: virtualenv [OPTIONS] DEST_DIR

### ISSUE THREAD

**Issue by imgauravj** [aws/aws-elastic-beanstalk-cli-setup#20]
I have installed from the git.  Now when I run .\aws-elastic-beanstalk-cli-setup\scripts\bundled_installer I get this error.--------------->

```

PS C:\WINDOWS\system32> cd E:\ebcli_demo
PS E:\ebcli_demo> .\aws-elastic-beanstalk-cli-setup\scripts\bundled_installer

==============================================
I. Installing Python
==============================================

**************************************************
1. Looking for existing Python 3.7.3 installation.
**************************************************
Python 3.7.3 is already installed.

************************
2. Installing virtualenv
************************
Collecting virtualenv
  Using cached https://files.pythonhosted.org/packages/ca/ee/8375c01412abe6ff462ec80970e6bb1c4308724d4366d7519627c98691ab/virtualenv-16.6.0-py2.py3-none-any.whl
Installing collected packages: virtualenv
Successfully installed virtualenv-16.6.0
You are using pip version 19.0.3, however version 19.1.1 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.

==============================================
II. Creating self-contained EBCLI installation
==============================================
Installing the EBCLI in C:\Users\Gaurav Jain\.ebcli-virtual-env

******************************************
1. Creating exclusive virtualenv for EBCLI
******************************************
There must be only one argument: DEST_DIR (you gave C:\Users\Gaurav Jain\.ebcli-virtual-env)
Usage: virtualenv [OPTIONS] DEST_DIR

Options:
  --version             show program's version number and exit
  -h, --help            show this help message and exit
  -v, --verbose         Increase verbosity.
  -q, --quiet           Decrease verbosity.
  -p PYTHON_EXE, --python=PYTHON_EXE
                        The Python interpreter to use, e.g.,
                        --python=python3.5 will use the python3.5 interpreter
                        to create the new environment.  The default is the
                        interpreter that virtualenv was installed with
                        (c:\program files\python37\python.exe)
  --clear               Clear out the non-root install and start from scratch.
  --no-site-packages    DEPRECATED. Retained only for backward compatibility.
                        Not having access to global site-packages is now the


```

**Comment by rahulrajaram** [comment]
Thanks for reporting your issue @imgauravj .

It is a problem in `scripts\ebcli_installer.py`. The installer is not handling the space in `"C:\Users\Gaurav Jain\.ebcli-virtual-env"` causing virtualenv to think two paths are being passed. I'm able to reproduce it. 

I'm going to release a fix for it shortly.

**Comment by rahulrajaram** [comment]
Closing this issue because it has been addressed in release v0.1.2. `git pull master` to get the latest changes.
